docs/version.rst

```diff
@@ -0,0 +1 @@
+|version|
```

Comment:
We're getting warnings about how this file isn't in the table of contents:
```
/srv/buildslave/flocker-docs/build/docs/version.rst:: WARNING: document isn't included in any toctree
```
e.g. http://build.clusterhq.com/builders/flocker-docs/builds/9718

Can you please either add it to the toctree or suppress the warning? (I'm afraid I don't know how to do the latter).